knative-operator/deploy/resources/dashboards/eventing/grafana-dash-knative-eventing-broker.yaml

```diff
@@ -118,7 +119,7 @@ data:
           "tableColumn": "",
           "targets": [
             {
-              "expr": "sum(rate(mt_broker_ingress_event_count{namespace_name=~\"$eventNamespace\"}[1m]))",
+              "expr": "sum(rate(mt_broker_ingress_event_count{namespace_name=~\"$namespace\"}[1m]))",
```

Comment:
Interesting could be that this is a convention, I think I hit that before but it is so long ago.

Reply:
Yes, when I applied them initially without this change, it was showing another select option as `eventNamespace` and when renamed to `namespace` it remove that option as it is taking it from the top level project select option